config.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Configuration.
+ *
+ * Project Configuration for gulp tasks.
+ *
+ * In paths you can add <<glob or array of globs>>. Edit the variables as per your project requirements.
+ */
+ 
+ module.exports = {
+
+	// START Editing Project Variables.
+	// Project related.
+	project                 : 'WPGulpTheme', // Project Name.
```

Comment:
What is this? This doesn't look like an JS `object` and neither is it a `var`, `let`, or `const`?

Reply:
To be honest, I was not 100% familiar with this way of working either - but it seems to be utilized in large projects that use Gulp.

This article does a good job of explaining it: https://www.sitepoint.com/understanding-module-exports-exports-node-js/

Reply:
I think this is done wrong! 

E.g. `project` is nothing. It should be a `const` and so does all variables that don't change the values. You seem to have removed the word `var` but haven't replaced it with either `let` or `const`. Go with `const`.

Reply:
No, it's not wrong. It's an export module. Check out the articles above.

Reply:
OK, that makes sense. I am just not sure what that is. As in it's not JavaScript object since JS objects have keys as strings which is not the case here. Though, it seems to be working here. Do you have any insights in there for me?

Reply:
I think this is good for now to be merged. Just the spacing and indentation issues for which I think we should add `eslint` and `prettier` configs.